book/performance.qmd

```diff
@@ -855,27 +865,84 @@ autoplot(bma, type = "cd")
 
 Similar to the test output before, this visualization leads to the conclusion that the `"featureless"` learner and `"ranger"` are significantly different, whereas the critical rank difference of 1.66 is not exceed for the comparison of the `"featureless"` learner, `"rpart"` and `"ranger"`, respectively.
 
-## ROC Analysis {#sec-roc}
+-->
+
+## Evaluation of Binary Classifiers {#sec-roc}
 
 <!-- So far we have focused on methods applicable to general classification and regression tasks. -->
 <!-- For the special case of binary classification, there are specialized performance measures and methods to analyze and compare the performance. -->
 <!-- Binary classification is unique because of the presence of a positive and negative class and a threshold probability to distinguish between the two. -->
-`r index("ROC")` (Receiver Operating Characteristic) analysis is widely used to evaluate binary classifiers.
-Although extensions for multiclass classifiers exist (see e.g., @hand2001simple), we will only cover the much easier binary classification case here.
-For binary classifiers that predict discrete classes, we can compute a `r index("confusion matrix")` which computes the following quantities (see also @fig-confusion):
 
-* **True positives (TP)**: Instances that are actually positive and correctly classified as positive.
-* **True negatives (TN)**: Instances that are actually negative and correctly classified as negative.
-* **False positives (FP)**: Instances that are actually negative but incorrectly classified as positive.
-* **False negatives (FN)**: Instances that are actually positive but incorrectly classified as negative.
+For the special case of binary classification, there are specialized performance measures and visualization tools to analyze and compare the performance of classifiers.
```

Comment:
There's specialised performance measures for all tasks. I think could be better just to say something like "We will now look at some specialized performance measures for binary classification that are implemented in mlr3"